### ISSUE TITLE
Feedback: hide warnings for 1.4.6 Contrast (Enhanced) by default

### ISSUE THREAD

**Issue by robinmetral** [webhintio/hint#6027]
## Description

When running a scan with default settings, webhint shows warnings for any color pairs that don't have a contrast ratio of 7:1. This is the criteria for WCAG's 1.4.6 Contrast (Enhanced), a AAA success criterion.

I'd have expected the scan to target 4.5:1 (1.4.3 Contrast (Minimum)). Shouldn't 1.4.6 violations be reported with severity=hint instead of severity=warning, at least? Currently results can be very noisy because of this, and perfectly good AA contrast ratios can bury the odd AA contrast issue.

## Environment

* __webhint version:__ 2.4.17 
* __Browser version:__ n/a 
* __URL for which webhint failed:__ n/a

## `webhint` configuration

<details>
<summary>webhint’s configuration</summary>

Categories:
* [x] Accessibility
* [x] Compatibility
* [x] Performance
* [x] Pitfalls
* [x] PWA
* [x] Security

Your target browsers:
* [x] Recommended settings
* [ ] Custom:  

Ignored resources:
* * [x] None
* * [ ] Different origin
* * [ ] Custom:  

</details>

## Debug output

n/a
